Userland/Libraries/LibEDID/DMT.cpp

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/String.h>
+#include <LibEDID/DMT.h>
+
+namespace LibEDID {
```

Comment:
With the exception of `LibDSP` (not sure why :thinking:), we don't use `namespace LibFoo`. `LibFoo` provides the namespace `Foo`.